.github/workflows/linux-check.yaml

```diff
@@ -50,7 +50,10 @@ jobs:
           sudo apt install -y \
               ninja-build \
               qtbase5-dev \
-              libqt5svg5-dev
+              libqt5svg5-dev \
+              clang-tools-11 \
+              libegl1-mesa \
+              libegl1-mesa-dev
```

Comment:
Should docs/INSTALL.md be updated? Is it feasible/does it worth it to make mesa a conditional dependency? What are pros and cons?

Reply:
I think I would remove the dependency on EGL, and then no change is needed here.

Reply:
For now all removed.